showcase/src/assets/translations/fr.json

```diff
@@ -154,6 +154,45 @@
         "SECTIONS": "Menu avec sections",
         "TITLE": "Stark app menu"
       },
+      "MESSAGE_PANE": {
+        "BUTTONS": {
+          "CENTER": "Centre",
+          "CHANGE_THEME": "Change le Thème",
+          "CLEAR_AND_SHOW_NEW_MESSAGES": "Effacer & Afficher les Nouveaux Messagess",
+          "CLEAR_MESSAGES": "Effacer les messages",
+          "LEFT": "Gauche",
+          "RIGHT": "Droite",
+          "SHOW_MESSAGES": "Afficher les Messages"
+        },
+        "DESCRIPTION": "La fenêtre de messages reste sur la page jusqu'à sa fermeture, mais les interactions de page ne sont pas bloquées (comme dans le cas d'une boîte de dialogue). Elle peut être utilisée pour afficher les erreurs de validation du formulaire si votre application à besoin de les conserver jusqu'à ce qu'elles soient rejetées.",
```

Comment:
Translation to adapt: 
* "DESCRIPTION": "Le volet de messages reste sur la page jusqu'à ce qu'il soit fermé. Néanmoins il ne bloque pas les interactions au sein de la page (contrairement aux boîtes de dialogue). Il peut, par exemple, être utilisé pour afficher les erreurs de validation d'un formulaire dans le cas où votre application nécessite de les conserver jusqu'à ce qu'elles soient fermées.",